app/services/shopify_import/importers/base_importer.rb

```diff
@@ -0,0 +1,31 @@
+module ShopifyImport
+  module Importers
+    class BaseImporter
+      def self.import!(**params)
```

Comment:
http://stackoverflow.com/questions/612189/why-are-exclamation-marks-used-in-ruby-methods

http://stackoverflow.com/questions/25293302/why-do-some-ruby-methods-need-a-bang-and-others-dont-to-be-a-destructive-method